wingtips-java8/src/main/java/com/nike/wingtips/util/AsyncWingtipsHelper.java

```diff
@@ -45,7 +45,7 @@
  *   AsyncWingtipsHelper asyncHelper = AsyncWingtipsHelper.DEFAULT_IMPL;
  *   Executor executor = Executors.newSingleThreadExecutor();
  *
- *   executor.execute(asyncHelper.runnableWithTracing(() -> {
+ *   executor.execute(asyncHelper.runnableWithTracing(() -&gt; {
```

Comment:
Misinterpreted some JavaDoc errors as reasons for build errors.  (The real reasons was I needed to increase branch coverage for new code added to RequestTracingFilter